ros_cross_compile/ros_cross_compile.py

```diff
@@ -143,6 +148,11 @@ def cross_compile_pipeline(
             'Specified workspace "{}" does not look like a colcon workspace '
             '(there is no "src/" directory). Cannot continue'.format(ros_workspace_dir))
 
+    metrics_dir = ros_workspace_dir / Path('cc_internals/metrics_data')
```

Comment:
let's make sure we don't write "cc_internals" more than one place in the entire program. It probably needs to be a constant that gets imported by `sysroot_creator` where it currently resides. Likewise the datacollector should use that relative path constant + the ros workspace to construct its own `metrics_data` path - we shouldn't do that here.

Reply:
also, don't use "/" in strings. `Path("cc_internals") / Path("metrics_data")`